docs/developer-guide/tasks/README.md

```diff
@@ -261,6 +294,66 @@ tasks:
 </div>
 
 
+### Fail
+
+This task will fail the flow; it can be used with or without conditions.
+
+Without conditions, it can be used, for example, to fail on some switch value.
+
+```yaml
+id: fail-on-switch
+namespace: io.kestra.tests
+
+inputs:
+  - name: param
+    type: STRING
+    required: true
+
+tasks:
+  - id: switch
+    type: io.kestra.core.tasks.flows.Switch
+    value: "{{inputs.param}}"
+    cases:
+      case1:
+        - id: case1
+          type: io.kestra.core.tasks.log.Log
+          message: Case 1
+      case2:
+        - id: case2
+          type: io.kestra.core.tasks.log.Log
+          message: Case 2
+      notexist:
```

Comment:
This example is wrong, if you don't have a default case and don't handle the case, the flow will fail anyway.
To make sense, you should have a default case. That way, you may decide to fail on a specific value and just let any other value pass.

Reply:
Well, it's not wrong but it's maybe too simplified.
I added a default, I'l also send a PR to Kestra as the same example is on the task documentation